doc/lsd.md

```diff
@@ -140,6 +140,9 @@ lsd is a ls command with a lot of pretty colours and some other stuff to enrich
 `--header`
 : Display block headers
 
+`--literal`
```

Comment:
according to https://man7.org/linux/man-pages/man1/ls.1.html

we can add `-N` as short name

Reply:
Should be fixed now!